HLTrigger/Configuration/python/HLTrigger_EventContent_cff.py

```diff
@@ -19,6 +19,7 @@
         'keep triggerTriggerEvent_*_*_*',
         'keep *_hltFEDSelectorL1_*_*',
         'keep *_hltScoutingEgammaPacker_*_*',
+        'keep *_hltScoutingMuonPacker_*_*',
```

Comment:
this file is auto-generated so it will be rewritten next time we derive new tables. 
I think if you want this wee need a CMSHLT JIRA (or you patch by hand the file in the `Configuration/EventContent`).